openhands/core/config/sandbox_config.py

```diff
@@ -73,7 +73,10 @@ class SandboxConfig(BaseModel):
     runtime_startup_env_vars: dict[str, str] = Field(default_factory=dict)
     browsergym_eval_env: str | None = Field(default=None)
     platform: str | None = Field(default=None)
-    close_delay: int = Field(default=15)
+    close_delay: int = Field(
+        default=int(os.environ.get('SANDBOX_CLOSE_DELAY_SECONDS', '3600')),
+        description='The delay in seconds before closing the sandbox after the agent is done. Can be configured via SANDBOX_CLOSE_DELAY_SECONDS env var.',
```

Comment:
we already override these with env vars, no need to introduce a new one here
```suggestion
        default=3600,
        description='The delay in seconds before closing the sandbox after the agent is done.',
```